__mocks__/min-delay-mock.js

```diff
@@ -0,0 +1 @@
+module.exports = () => new Promise(resolve => setTimeout(resolve, 1000));
```

Comment:
```suggestion
module.exports = () => new Promise(resolve => setTimeout(resolve, 1000));

```
you need to add an empty line at the end of the file